src/malcolm/reducer/navigation.reducer.js

```diff
@@ -103,13 +103,10 @@ function updateNavTypes(state) {
             nav.path.split('.')[0]
           );
 
-          if (matchingAttribute) {
+          if (matchingAttribute && !matchingAttribute.calculated.loading) {
```

Comment:
My first thought about the failing cypress test is it might be caused by one of the changes in this reducer.